### DISCUSSION TITLE
Value for color doesn't work correctly.

### DISCUSSION THREAD

**Discussion by cgkaren** [inlife/nexrender#906] [Q&A] [answered]
When I write numbers in a value, it only works if one of them is zero, but if you give a value for all three colors, then it renders emptiness, can you help ? 

Тhanks

**Comment by inlife** [comment]
Hey @cgkaren 

Could you provide a sample of how do the values you set up look like?

Cheers

**Comment by cgkaren** [comment]
{
"type": "data",
"composition": "main",
"layerName": "text-here",
"property": "Effects.Fill.Color",
"value": [35, 200, 32]
},

**Comment by cgkaren** [comment]
Thanks

**Comment by cgkaren** [comment]
But when I write like this, it renders
{
"type": "data",
"composition": "main",
"layerName": "text-here",
"property": "Effects.Fill.Color",
"value": [35, 0, 32]
},

or

{
"type": "data",
"composition": "main",
"layerName": "text-here",
"property": "Effects.Fill.Color",
"value": [0, 35, 32]
},


With the best,
Karen

**Comment by inlife** [comment, marked as answer]
For colors in After Effects, make sure all color values are in the range from 0..1 as a floating point number.

So easiest way: you can divide each number in your set by 255 to achieve that.

Example: "value": [0.137, 0.784, 0.125]

**Comment by cgkaren** [comment]
Can you show at this example? If it's not difficult for you, thank you very much!

"value": [35, 200, 32]

**Reply by inlife** [reply]
"value": [0.137, 0.784, 0.125]

**Reply by cgkaren** [reply]
Thank you very much! It works.

**Comment by cgkaren** [comment]
I'm writing but it still doesn't load :/

**Reply by JamesBotterill** [reply]
I find a better way of working with colour within nexrender is to use this helpful package (https://github.com/dberget/nexrender-action-normalize-color)
